commit cb1d8a8e75f203d97a2f233e55232f3e14496581
Author: Nick Leverton <nick@leverton.org>
Date:   Wed Aug 6 23:22:37 2014 +0100

    Add documentation for the smtp, qmqp and smtpd modules
    (see also debian/man).

diff --git a/doc/nullmailer-send.8 b/doc/nullmailer-send.8
index 7fa419b..5c2a767 100644
--- a/doc/nullmailer-send.8
+++ b/doc/nullmailer-send.8
@@ -130,6 +130,6 @@ The protocol program directory.
 nullmailer-queue(8),
 nullmailer-inject(1),
 mailq(1),
-/usr/local/libexec/nullmailer/smtp --help,
-/usr/local/libexec/nullmailer/qmqp --help
+nullmailer-smtp(8),
+nullmailer-qmqp(8),
 
